From https://github.com/cookeda/CapstoneSportsbook
 * branch            main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 235ebc2] Automated Server Update
 2 files changed, 13 insertions(+)
 create mode 100644 dailyserverlogfile.log
To https://github.com/cookeda/CapstoneSportsbook.git
   983b708..235ebc2  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch            main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("./Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: './Data Collecting Src'
[main b22e7c9] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   78a75c1..b22e7c9  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch            main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 6f86681] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   b48dd0e..6f86681  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch            main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main b1f0102] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   0035e5f..b1f0102  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch            main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 300a56d] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   cc9fc99..300a56d  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch              main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 77de77da] Automated Server Update
 2 files changed, 14 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   34b14411..77de77da  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch              main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 7186f7f5] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   288e32a0..7186f7f5  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch              main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main d5aa4cb5] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   76186943..d5aa4cb5  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch              main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 1e3c28bf] Automated Server Update
 1 file changed, 9 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   e50d317a..1e3c28bf  main -> main
From https://github.com/cookeda/CapstoneSportsbook
 * branch              main       -> FETCH_HEAD
Already up to date.
Traceback (most recent call last):
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 58, in <module>
    main()
  File "/media/myfiles/CapstoneSportsbook/Daily.py", line 12, in main
    os.chdir("media/myfiles/CapstoneSportsbook/Data Collecting Src")
FileNotFoundError: [Errno 2] No such file or directory: 'media/myfiles/CapstoneSportsbook/Data Collecting Src'
[main 7571386c] Automated Server Update
 2 files changed, 13 insertions(+)
To https://github.com/cookeda/CapstoneSportsbook.git
   ee88608a..7571386c  main -> main
